### PR TITLE
feat [SRTP-1678]: add tests optional EPC fields

### PR DESCRIPTION
**Pull request by michele-fucito**
### Description

This PR adds functional test coverage for optional EPC fields in the RTP send flow, specifically validating how the system handles edge cases in the synchronous response from the debtor service provider mock.

Two new mock fiscal codes are introduced to drive specific scenarios:
- A fiscal code that triggers an ACTC response **without** the `_links` field
- A fiscal code that triggers an ACTC response **with an unexpected extra field**

The corresponding env vars are wired up in the GitHub Actions workflow and documented in the README.

### List of changes

- Added `test_send_rtp_sync_accepted_no_links`: verifies that an RTP reaches status `ACCEPTED` when the EPC synchronous response omits the optional `_links` field
- Added `test_send_rtp_sync_sent_extra_field`: verifies that an RTP remains in status `SENT` when the EPC synchronous response includes an unknown extra field (i.e., the system does not fail on unexpected fields)
- Added `MOCK_NO_LINKS_FISCAL_CODE` and `MOCK_EXTRA_FIELD_FISCAL_CODE` to `config/configuration.py` and `.github/workflows/run_tests.yml`
- Documented the two new env vars in `README.md`

### Motivation and context

The EPC standard defines some fields as optional in the synchronous response. The system should handle both their absence and the presence of unknown fields gracefully. These tests ensure the RTP state machine behaves correctly in both edge cases without breaking existing flows.

### Aggregation level

- [ ] Test case
- [x] Test suite

### Test type

- [x] Functional test
- [ ] Bdd test
- [ ] Performance
- [ ] End to end
- [ ] Performance

### Type of changes

- [ ] Add new test case/suite
- [x] Modify existing test case/suite

### Test Results

- [x] Success
- [ ] Failure
- [ ] Poor performance
- [ ] Good performance

### Did you update secrets accordingly?

- [ ] Yes
  - [ ] `.secrets_template.yaml`
  - [ ] Pipelines' secure file
  - [ ] Confluence
- [ ] Not needed
- [ ] No (_please provide further information_)

### Did you update dependencies accordingly?

- [ ] Yes (which requirements.txt did you update?)
- [x] Not needed

### CODE REVIEW

**Comment by fmuscianisi** [comment]
Maybe we also need test cases for synchronous `RJCT` to cover all of our use cases.

File: functional-tests/tests/send_rtp/test_RTP_send_MOCK.py [hunk 1]
```diff
@@ -173,3 +173,79 @@ def test_send_rtp_sync_rejected_ds08p_n(
...
+
+    location = send_response.headers["Location"]
+    resource_id = location.split("/")[-1]
+
+    get_response = get_rtp(
+        access_token=rtp_reader_access_token,
+        rtp_id=resource_id,
+    )
+    assert get_response.status_code == 200
+    assert get_response.json()["status"] == "SENT"
```
[the remark this answers is not in the document]
Comment (michele-fucito) [reply functional-tests/tests/send_rtp/test_RTP_send_MOCK.py]: done

File: functional-tests/tests/send_rtp/test_RTP_send_MOCK.py [hunk 2]
```diff
@@ -173,3 +173,79 @@ def test_send_rtp_sync_rejected_ds08p_n(
...
+
+    location = send_response.headers["Location"]
+    resource_id = location.split("/")[-1]
+
+    get_response = get_rtp(
+        access_token=rtp_reader_access_token,
+        rtp_id=resource_id,
+    )
+    assert get_response.status_code == 200
+    assert get_response.json()["status"] == "ACCEPTED"
```
[the remark this answers is not in the document]
Comment (michele-fucito) [reply functional-tests/tests/send_rtp/test_RTP_send_MOCK.py]: done

File: functional-tests/tests/send_rtp/test_RTP_send_MOCK.py [hunk 3]
```diff
@@ -173,3 +173,79 @@ def test_send_rtp_sync_rejected_ds08p_n(
...
+
+
+@allure.epic("RTP Send")
+@allure.feature("RTP Send")
+@allure.story("Service provider sends an RTP with synchronous ACTC response containing an unexpected field")
+@allure.title("An RTP sent when EPC response contains an unknown field is in status SENT")
```
[the remark this answers is not in the document]
Comment (michele-fucito) [reply functional-tests/tests/send_rtp/test_RTP_send_MOCK.py]: done

**Review by fmuscianisi — APPROVED** [review]
